### PR TITLE
IInstructions for contour

### PR DESCRIPTION
**Pull request by SimpleVlad**

### CODE REVIEW

File: curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/contour.md [hunk 1]
```diff
@@ -61,10 +64,12 @@ tests:
 <h2>OpenCV.js</h2>
 <img id="imageSrc" src="https://s3.amazonaws.com/freecodecamp/FCCStickers-CamperBot200x200.jpg"/>
 <p></p>
+<canvas id="canvasCanny" ></canvas>
+<p></p>
 <canvas id="canvasOutput" ></canvas>
 
 <script async src="https://docs.opencv.org/master/opencv.js"
-        onload='cv["onRuntimeInitialized"]=()=> { contour() }'
+        onload='cv["onRuntimeInitialized"]=()=> { runSample() }'
```
Comment (dkurt) [inline curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/contour.md:68]: Maybe just `run`?

Comment (SimpleVlad) [reply curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/contour.md:68]: IDN,  but i think <code>runSample()</code> is more informative.

Comment (dkurt) [reply curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/contour.md:68]: Ok, let's keep `runSample`

File: curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/contour.md [hunk 2]
```diff
@@ -7,21 +7,14 @@ videoUrl:
...
-First, you'll start by building a simple web page using HTML. You can edit <code>code</code> in your <code>code editor</code>, which is embedded into this web page.
-Do you see the code in your code editor that says <code>&#60;h1&#62;Hello&#60;/h1&#62;</code>? That's an HTML <code>element</code>.
-Most HTML elements have an <code>opening tag</code> and a <code>closing tag</code>.
-Opening tags look like this:
-<code>&#60;h1&#62;</code>
-Closing tags look like this:
-<code>&#60;/h1&#62;</code>
-The only difference between opening and closing tags is the forward slash after the opening bracket of a closing tag.
-Each challenge has tests you can run at any time by clicking the "Run tests" button. When you pass all tests, you'll be prompted to submit your solution and go to the next coding challenge.
+Contours can be explained simply as a curve joining all the continuous points (along the boundary), having same color or intensity. The contours are a useful tool for shape analysis and object detection and recognition. But firstly in this example we use <code>cv.Canny</code> to find edge of object.And only after that we can find a conturs.
```
Comment (dkurt) [inline curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/contour.md]: edge -> edges

Comment (dkurt) [reply curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/contour.md]: "a conturs" -> "contours"

Comment (dkurt) [reply curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/contour.md]: There are still "conturs"

Comment (SimpleVlad) [reply curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/contour.md]: <code>And only after that we can find contours </code> write in "files changed"

Comment (dkurt) [reply curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/contour.md]: @SimpleVlad, 
> We use functions: <code>cv.findContours (image, contours, mode, method) </code> where <code>image</code> is an input image,<code>conturs</code> is detect contours, each contour is stored as a vector of points <code>mode</code> is contour retrieval mode, <code>method</code> is contour approximation method, and <code>cv.drawContours(image,  contours,  contourIdx, color, thickness)</code> where <code>image</code> is an output image,<code>conturs</code> is all input conturs,<code>contourIdx</code> is a  parameter indicating a contour to draw, <code>color</code> is a color of the contours, <code>thickness</code> is a thickness of lines.

File: curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/contour.md [hunk 3]
```diff
@@ -47,17 +40,19 @@ tests:
...
 
     cv.Canny(mat, edged, 100, 200);
 
-    cv.findContours(edged, contours, hierarchy, cv.RETR_CCOMP, cv.CHAIN_APPROX_SIMPLE);
+    cv.imshow("canvasCanny", edged);
 
-    cv.drawContours(mat, contours, -1, new cv.Scalar(0, 255, 0, 255));
+   
+
+    cv.drawContours(mat, contours, -1, new cv.Scalar(255, 0, 0, 255),2);
```
Comment (dkurt) [inline curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/contour.md]: What

File: curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/contour.md [hunk 4]
```diff
@@ -47,17 +40,17 @@ tests:
...
     let hierarchy = new cv.Mat();
 
     cv.Canny(mat, edged, 100, 200);
+    cv.imshow("canvasCanny", edged);
 
-    cv.findContours(edged, contours, hierarchy, cv.RETR_CCOMP, cv.CHAIN_APPROX_SIMPLE);
-
-    cv.drawContours(mat, contours, -1, new cv.Scalar(0, 255, 0, 255));
+   
+    cv.drawContours(mat, contours, -1, new cv.Scalar(255, 0, 0, 255),2);
```
Comment (dkurt) [inline curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/contour.md]: Let's ask user to implement this function if you don't mind.

Comment (SimpleVlad) [reply curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/contour.md]: Sorry,  I copy this line, but don't delete

**Review by dkurt — APPROVED** [review]
:+1:

**Pull request merged**